imap_processing/swe/tests/test_decom.py

```diff
@@ -1,3 +1,5 @@
+import os
```

Comment:
We should stick with pathlib throughout the library for consistency.
```suggestion
import pathlib
```